src/engine/enginebuffer.cpp

```diff
@@ -749,7 +744,11 @@ void EngineBuffer::process(CSAMPLE* pOutput, const int iBufferSize) {
     // We do this even if rubberband is not active.
     if (sample_rate != m_iSampleRate) {
         if (m_pScaleRB != NULL) {
-            m_pScaleRB->initializeRubberBand(sample_rate);
+            // TODO: this dynamic cast will be removed once we refactor scalers
+            // to all have a set-sample-rate member.  For now it's ok because
+            // this is called very rarely.
+            dynamic_cast<EngineBufferScaleRubberBand*>(m_pScaleRB)->initializeRubberBand(
```

Comment:
use static cast, if you are sure that it will not fail or check for NULL after conversion